tracks/javascript/exercises/space-age/mentoring.md

```diff
@@ -28,14 +28,14 @@ key or a function to map from that name to the orbital period is preferred:
 ```javascript
 function getOrbitalPeriod(planet) {
   const [p, ...lanet] = planet
-  return ORBITAL_PERIODS[[p.toUpperCase(), ...lanet].join('')]
+  return ORBITAL_PERIODS[[p.toUpperCase(), ...planet].join('')]
```

Comment:
This code now does something else because both `lanet` and `planet` are existing variables in current scope 🤔. Also looking at the previous like, I think `lanet` might be on purpose, like a pun? `[p, ...lanet] = planet`?

Reply:
Angelika thanks for catching that!

Reply:
This was not a typo:

```javascript
[p.toUpperCase(), ...lanet].join('')
```

Generates:

```shell
Planet
```

Reply:
Perhaps this would be clearer:

```javascript
function getOrbitalPeriod(planet) {
  const [prefix, ...rest] = planet
  return ORBITAL_PERIODS[[prefix.toUpperCase(), ...planet].join('')]
}
```

What do you think?